packages/create-next-app/helpers/git.ts

```diff
@@ -30,6 +30,8 @@ export function tryGitInit(root: string): boolean {
     execSync('git init', { stdio: 'ignore' })
     didInit = true
 
+    execSync('git checkout -b main', { stdio: 'ignore' })
```

Comment:
by doing this the repo will end up having two branches right? why not just rename the branch?

```
git branch -m main
```